features/aanschrijfwijze.feature

```diff
@@ -136,135 +141,622 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
+      | geslachtsnaam (02.40)          | Groen              |
+      | voornamen (02.10)              | Ferdinand Cornelis |
+      | aanduiding naamgebruik (61.10) | V                  |
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992935 |
+      En de partner heeft de volgende naam gegevens
+      | naam                  | waarde |
+      | voorvoegsel (02.30)   |        |
+      | geslachtsnaam (02.40) | Geel   |
+      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+      | naam                  | waarde   |
+      | datum aangaan (06.10) | 19580401 |
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992936 |
+      En de partner heeft de volgende naam gegevens
+      | naam                  | waarde |
+      | voorvoegsel (02.30)   |        |
+      | geslachtsnaam (02.40) | Roodt  |
+      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+      | naam                  | waarde   |
+      | datum aangaan (06.10) | 19610808 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | naam.aanschrijfwijze            |
+      Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+      | naam            | waarde          |
+      | aanschrijfwijze | F.C. Geel-Groen |
+
+  Rule: Bij meerdere huwelijken/partnerschappen die allen ontbonden zijn, wordt naam gegevens van de laatst ontbonden partner gebruikt voor het samenstellen van de aanschrijfwijze
+
+    Scenario: meerdere ontbonden relaties gebruikt de laatst ontbonden relatie
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992934 |
+      En de persoon heeft de volgende naam gegevens
+      | naam                           | waarde |
+      | voorvoegsel (02.30)            |        |
+      | geslachtsnaam (02.40)          | Wit    |
+      | voornamen (02.10)              | Jan    |
+      | aanduiding naamgebruik (61.10) | V      |
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992935 |
+      En de partner heeft de volgende naam gegevens
+      | naam                  | waarde |
+      | voorvoegsel (02.30)   |        |
+      | geslachtsnaam (02.40) | Geel   |
+      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+      | naam                     | waarde   |
+      | datum aangaan (06.10)    | 19580401 |
+      | datum ontbinding (07.10) | 19601001 |
```

Comment:
datum ontbinding zit niet in aangaanHuwelijkPartnerschap, maar in ontbindingHuwelijkPartnerschap